File: Review/C/review-6.c
#pragma warning(disable:4996)
#include <stdio.h>

typedef struct Student {
    char name[10];
    int score;
}Student;

int main() {

    Student student[5];
    int total = 0;
    for(int i = 0; i < 5; i++) {
        scanf("%s %d", student[i].name, &student[i].score);
        total += student[i].score;
    }

    int avg = total / 5.0;
    for(int i = 0; i < 5; i++) {
        if(avg >= student[i].score)
            printf("%s\n", student[i].name);
    }

    return 0;
}

File: 1-Review/Python/review-6.py
class Student:
    def __init__(self, name, score):
        self.name = name
        self.score = score

def main():
    students = []
    total = 0

    for _ in range(5):
        name, score = input().split()
        score = int(score)
        students.append(Student(name, score))
        total += score

    avg = total / 5.0

    for student in students:
        if avg >= student.score:
            print(student.name)

if __name__ == '__main__':
    main()